hack/make-pjs.sh

```
@@ -0,0 +1,191 @@
+  [ -z "$CLUSTER" ] && echo "CLUSTER is required" && exit 1
+  [ -z "$HOW_MANY" ] && echo "HOW_MANY is required" && exit 1
+  [ -z "$TYPE" ] && echo "TYPE is required" && exit 1
+
+  case "$TYPE" in
+      "e2e")
+           PROWJOBS_CONFIG="${PROWJOBS_CONFIG:-ci-operator/jobs/openshift/installer/openshift-installer-main-presubmits.yaml}"
+           extract_test_info_from_job "$PROWJOBS_CONFIG"
+           dump_test_info
+           tests=$(e2e_tests "$PROWJOBS_CONFIG" | sort -R)
+           [ -z "$tests" ] && echo "no e2e tests found in ${RELEASE_REPO}/${PROWJOBS_CONFIG}" && exit 1
+           top_n=$(head -n $HOW_MANY <<<"$tests")
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)